i18n/ar.toml

```
@@ -1,80 +1,91 @@
 # UI strings. Buttons and similar.
 
 [ui_pager_prev]
-other = "السابق"
+other = "السَّابق" 
 
 [ui_pager_next]
-other = "التالي"
+other = "التَّالي" 
 
 [ui_read_more]
-other = "إقرأ المزيد"
+other = "اِقْرأ اَلمزِيد" 
 
 [ui_search]
-other = "ابحث في هذا الموقع"
+other = "اِبْحث فِي هذَا اَلموْقِع" 
 
 # Used in sentences such as "Posted in News"
 [ui_in]
-other = "في"
+other = "فِي" 
+
 # Used in sentences such as "All Tags"
 [ui_all]
-other = "كل"
+other = "كُلَّ" 
 
-# Footer text
+# Footer text 
 [footer_all_rights_reserved]
-other = "كافة الحقوق محفوظة"
+other = "كَافَّة اَلحُقوق مَحفُوظة" 
 
 [footer_privacy_policy]
-other = "سياسة الخصوصية"
-
+other = "سِياسة الخصوصيَّة" 
 
-# Post (blog, articles etc.)
+# Post ( blog , articles etc . ) 
 [post_byline_by]
-other = "الكاتب"
+other = "الكاتب" 
 [post_created]
-other = "منشئة"
+other = "مُنشأ" 
 [post_last_mod]
-other = "اخر تعديل"
+other = "آخر تَعدِيل" 
 [post_edit_this]
-other = "عدل هذه الصفحة"
+other = "عدل هَذِه الصَّفْحة" 
 [post_view_this]
-other = "أنظر مصدر الصفحة"
+other = "أَنظُر مَصدَر الصَّفْحة" 
 [post_create_child_page]
-other = " أنشئ صفحة فرعية"
+other = "أَنْشِئ صَفحَة فَرعِية" 
 [post_create_issue]
-other = "أنشئ مسألة حول الوثائق"
+other = "أَنْشِئ مَسْأَلة حَوْل الوثائق" 
 [post_create_project_issue]
-other = "أنشئ مسألة حول المشروع"
+other = "أَنْشِئ مَسْأَلة حَوْل المشْروع" 
 [post_posts_in]
-other = "منشور في"
+other = "مَنشُور فِي" 
 [post_reading_time]
-other = "دقيقة للقراءة"
+other = "دَقِيقَة لِلْقراءة" 
 [post_less_than_a_minute_read]
-other = "أقل من دقيقة"
+other = "أقلَّ مِن دَقِيقَة" 
 
-# Print support
+# Print support 
 [print_printable_section]
-other = "هذا العرض يتضمن عدة صفحات للطباعة ضمن هذا القسم."
+other = "هذَا العرْض يَتَضمَّن عِدَّة صَفَحات لِلطِّباعة ضِمْن هذَا القسْم." 
 [print_click_to_print]
-other = "اضغط هنا للطباعة"
+other = "اِضْغط هُنَا لِلطِّباعة" 
 [print_show_regular]
-other = "العودة للعرض العادي"
+other = "اَلْعَودة لِلْعرْض العاديِّ" 
 [print_entire_section]
-other = "اطبع القسم كاملا"
+other = "اِطْبع القسْم كاملا" 
 
-# Community
+# Community 
 [community_join]
-other = "انضم الي مجتمع {{ .Site.Title }}"
+other = "اِنْضمَّ إِلى مُجتَمَع { { . Site . Title } }" 
```

Why all the extra spaces here? Remove the extra spaces added around the periods and braces, here and in other lines:

```suggestion
other = "اِنْضمَّ إِلى مُجتَمَع {{.Site.Title }}"
```

@deining ☝️

---

I think I'll have to review my editor settings. Will resolve it and update the request.